nextcloud/text
# Add Cypress tests for image insertion

refs #1900
fixes #2049
Tests:
* Insert image from Files
* Insert image from a link
* Upload a local image

The logic is to:
* Click on the menu action
* Perform the specific extra actions to select a file or upload a file or provide a link
* Intercept the network request to get the created file name
* Check the image is visible in the editor

A command to delete users has been added. It makes it more convenient for those who are not using a temporary NC instance to run the tests :grin:.
